### DISCUSSION TITLE
Documentation post install

### DISCUSSION THREAD

**Discussion by bstall23** [facileManager/facileManager#588] [General]
fM Version : 3.4.2
fmDNS Version : 3.3.4

I was able to get fmDNS working in a test environment by enabling the basic settings and modifying a test zone. The tool works great in that regards.

However, in order for me to use this in a practical application, I need to be able to connect to a remote bind DNS server to manage the zones there. The problem that I keep running into is that I don't know how to do this because the README file only covers up to the installation of the tool, there is nothing on proper management and deployment. I've read the 'Help' section but that just offers terminology definitions. As a fellow sysadmin, you can appreciate well maintained documentation so I hope you see my point.

**My feature request is for post installation documentation, specifically around the remote management of DNS servers and importing existing zones in for management under fmDNS.**

This is where I'm currently stuck so any guidance would be greatly appreciated. If this documentation already exists, please kindly point me to it

**Comment by WillyXJ** [comment]
Agreed and duplicate of #307.  Help is requested to write documentation, but a quick search on Google gave me https://mahedi.me/blog/install-dns-server-with-fmdns-graphical-tool/ which might get you going.

**Comment by WillyXJ** [comment]
The beginning of the official documentation is up!

https://docs.facilemanager.com/
